src/utils.ts

```diff
@@ -304,3 +313,33 @@ export function resolvePathPlaceHolders(path: string, folder?: WorkspaceFolder)
   }
   return path;
 }
+
+export function resolvePATHPlaceHolders(path: string, folder?: WorkspaceFolder) {
+  return path
+    .replace('${HOME}', os.homedir)
+    .replace('${home}', os.homedir)
+    .replace('$PATH', process.env.PATH!)
+    .replace('${PATH}', process.env.PATH!);
+}
```

Comment:
I don't understand your question. `resolvePATHPlaceHolders` is not the same as `resolvePathPlaceHolders`.

Reply:
I did not even consciously notice that there are two functions with an almost identical name...

Why are there two very similar functions, and this one has an unused parameter?

Reply:
> Why are there two very similar functions, and this one has an unused parameter?

Because this function replaces `PATH`, which is not a generic **path** variable, so they do different things.

Reply:
Maybe we don't even need it at all and can pass it unchanged. I haven't tested.

Reply:
How do we test whether we need it?